define_strong_self.m

```diff
@@ -0,0 +1,10 @@
+// Define Strong Self
+// 
+//
+// IDECodeSnippetCompletionPrefix: vokStrongSelf
+// IDECodeSnippetCompletionScopes: [CodeBlock]
+// IDECodeSnippetIdentifier: 1A45777F-3BA7-409C-BFA2-7B8FBCF20C58
+// IDECodeSnippetLanguage: Xcode.SourceCodeLanguage.Objective-C
+// IDECodeSnippetUserSnippet: 1
+// IDECodeSnippetVersion: 2
+__typeof__(self) strongSelf = weakSelf;
```

Comment:
* In terms of ```__typeof__``` vs ```typeof``` - I don't have a good reason except the sources I was reading had it defined that way.  But since according to [this](http://stackoverflow.com/questions/14877415/difference-between-typeof-typeof-and-typeof-objective-c) they are all the same.
* That was just an oversight on my part.

Both are great points and I'll update it accordingly for the next pr.